classes/classes/Creature.as

```diff
@@ -1984,7 +1984,12 @@ package classes
 			return _wingType == 2 || _wingType == 7 || _wingType == 9 || _wingType == 11 || _wingType == 12;
 
 		}
-		
+
+		public function canUseStare():Boolean
+		{
+			return eyeType == EYES_BASILISK;
+		}
+
```

Comment:
Basilisk eyes only for now, but in the future, we may have more similar eyes, like cockatrice eyes for example.